addons/rating/views/rating_templates.xml

```diff
@@ -62,7 +62,7 @@
                                            t-attf-id="rate_{{rate_name}}"
                                            t-att-value="rate_name"
                                            t-att-checked="rate == rate_name"/>
-                                    <label t-attf-class="col p-4 btn o_rating_label shadow-none transition-base {{rate == rate_name and 'active' or ''}}"
+                                    <label t-attf-class="col p-4 o_rating_label shadow-none transition-base {{rate == rate_name and 'active' or ''}}"
```

Comment:
Fix looks good! Thank you.

Can you add an explanation of why this fix is applied to 18.0 ? The class was not there before ? Mention the commit that causes the issue.

Reply:
Also this will not fix the issue for existing database, unless they update their module ...

Reply:
> Also this will not fix the issue for existing database, unless they update their module ...

Yeah most of the time in odoo for code changes it will require them to update right? of course there is another way to fix this which is directly change the boostrap btn css, but I am afraid that will influence other stuff, as it is a overall css change

Reply:
you could remove the border by changing the css rules for `.o_rating_label {` and remove the btn class in the master fw-port :+1:

Reply:
> [22c3cc5](https://github.com/odoo/odoo/commit/22c3cc51eda7f126178d70f765f846e411fe2d0a)
> 
> This is the original commit

add it to your commit message please :)